data/json/monsters/zed-classic.json

```diff
@@ -456,7 +456,7 @@
     "death_drops": "default_zombie_death_drops",
     "burn_into": "mon_zombie_scorched",
     "fungalize_into": "mon_zombie_tough_fungal",
-    "upgrades": { "half_life": 30, "into_group": "GROUP_ZOMBIE_UPGRADE" },
+    "upgrades": { "half_life": 24, "into": "mon_zombie_bruiser" },
```

Comment:
This will decrease zombie variety, potentially significantly, as all tough zomxies or anything that involves into them are forced down the bruiser->brute path.

This makes brutes _more_ common, not less.

Reply:
Oh yeah that is going to be mega annoying late game.

Reply:
Fixed by #71440